packages/expo-asset/src/AssetHooks.ts

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react';
+
+import { Asset } from './Asset';
+
+/**
+ * Download and store one or more assets locally.
```

Comment:
```suggestion
 * Downloads and stores one or more assets locally.
```

Reply:
[asset] Finetune docs in useAssets hooks